### PR TITLE
grammar issues, count

### PR DESCRIPTION
**Pull request by mk625sn**

### CODE REVIEW

**Comment by mk625sn** [comment]
the new merge request. Grammar issue - solved

**Comment by alopezlago** [comment]
I'll try to take a look at this by this weekend, if not earlier.

File: QuizBowlDiscordScoreTracker/QuizBowlDiscordScoreTracker.csproj [hunk 1]
```diff
@@ -35,16 +35,6 @@
     <PackageReference Include="Serilog.Sinks.File" Version="4.1.0" />
   </ItemGroup>
   
-  <ItemGroup>
```
Comment (alopezlago) [inline QuizBowlDiscordScoreTracker/QuizBowlDiscordScoreTracker.csproj:38]: Don't remove this. It's needed for easily testing dev changes.

File: QuizBowlDiscordScoreTracker/TeamManager/ITeamManager.cs [hunk 2]
```diff
@@ -12,5 +12,6 @@ public interface ITeamManager
         Task<string> GetTeamIdOrNull(ulong userId);
 
         Task<IReadOnlyDictionary<string, string>> GetTeamIdToNames();
+
```
Comment (alopezlago) [inline QuizBowlDiscordScoreTracker/TeamManager/ITeamManager.cs:15]: Nitpick: remove extra newline

File: QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace QuizBowlDiscordScoreTracker.TeamManager
+{
+    public interface IByRoleTeamManager
+    {
+        public void ReloadTeamRoles(out string message);
```
Comment (alopezlago) [inline QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs:10]: Nitpick: "public" isn't needed here, since it's an interface

**Review by alopezlago — CHANGES_REQUESTED** [review]
Just a couple minor fixes, and keeping the discordToken/config copy in the csproj file.

File: QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs (hunk 3 above)
Comment (mk625sn) [reply QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs:10]: solved

File: QuizBowlDiscordScoreTracker/discordToken.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿NzY0NjExNjc1NTIwNzYxODk2.X4IyHQ.PN6QNO0zTiMaxbVy9kaC0zLVreI
```
Comment (alopezlago) [inline QuizBowlDiscordScoreTracker/discordToken.txt:1]: Remove this file and config.txt from the brnach. It should be tracked under .gitignore, but if it's not, we should update that file to exclude these two.
You should also rotate this secret; you can generate a new one from the Discord portal.

File: QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs (hunk 3 above)
Comment (alopezlago) [reply QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs:10]: The interface itself should be public. The method doesn't need to be public.

**Review by alopezlago — CHANGES_REQUESTED** [review]

File: QuizBowlDiscordScoreTracker/discordToken.txt (hunk 4 above)
Comment (mk625sn) [reply QuizBowlDiscordScoreTracker/discordToken.txt]: such a stupid mistake... now it should be ok.

**Pull request merged**